Makefile

```diff
@@ -38,6 +39,7 @@ clean:
 	@-rm -rf public/components
 	@-rm -rf public/css
 	@-rm -rf public/urth_components
+	@-docker rmi -f $(KG_IMAGE_NAME) $(DASHBOARD_IMAGE_NAME)
```

Comment:
It is, it just seemed incongruous that it builds the images but never removes them.